codecarbon/external/logger.py

```diff
@@ -1,18 +1,61 @@
 import logging
 import os
+from typing import Optional
+
+
+def set_logger_format(custom_preamble: Optional[str] = ""):
+    logger = logging.getLogger("codecarbon")
+    format = f"[%(name)s %(levelname)s @ %(asctime)s]{custom_preamble} %(message)s"
```

Comment:
Space or brackets could be added here for custom_preamble..

Reply:
sold, adding brackets